corehq/apps/hqwebapp/widgets.py

```diff
@@ -38,7 +39,7 @@ def render(self, name, value, attrs=None):
         if value not in ('', True, False, None):
             # Only add the 'value' attribute if a value is non-empty.
```

Comment:
Should probably be `(u'', b'', True, False, None) then, because b'' != u'' ... unless value is definitely not a bytestring.

Reply:
Hmmm ... `Widget.format_value()` seems to assume either that `value` is Unicode ... or that `b'' == '' == u''` for both Python2 (true) and Python3 (false). ... So maybe this is OK as it is.